### PR TITLE
Removes war ops from the game

### PR DESCRIPTION
**Pull request by Yenwodyah**
## About The Pull Request

Removes war ops

## Why It's Good For The Game

War ops is boring and unfun. It turns the game into even more of a TDM than revs does and it removes any possibility of roleplaying and paranoia. Every round is exactly the same: run to armory and grab guns, grab all access, wait for ops to arrive, round ends without you firing a single bullet cause the ops landed on the opposite side of the station from you. Also, it has a freaking 20 MINUTE setup time when the station and the ops both have to sit around twiddling their thumbs. You could fit in two or three rounds of wizard in that time.

Closes #49279 

## Changelog
:cl:
del: Pitifully low nuke op success rates have convinced the Syndicate to stop giving out free money when head operatives give up their element of surprise for no reason.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
I’m 👍 on principle bc fuck war ops, but as far as I can tell this PR does nothing to make stealth/blitz (namely blitz) less of an unbalanced clusterfuck so its bad

**Comment by Yenwodyah** [comment]
@wesoda25 I agree that stealth ops is unbalanced, but I think that's partially because war ops dominates the gamemode so much that people don't care enough to discuss/PR stealth ops changes. Hopefully removing war ops puts more of a focus on stealth ops' issues.
If you have any specific concerns I'm up for making another PR to fix them.

**Comment by wesoda25** [comment]
Fair enough

**Comment by BadSS13Player** [comment]
oranges already tried to do this

**Comment by zxaber** [comment]
War ops is the only way to make nuke ops fun at all.

Just remove the whole mode.

**Comment by itseasytosee** [comment]
remove traitor

**Comment by QuarianCommando** [comment]
Imagine a playerbase being this unrobust.

**Comment by MrDoomBringer** [comment]
Cringe pr

**Comment by optimumtact** [comment]
I can only agree with OP, who is obviously incredibly handsome, talented and smart

**Comment by itseasytosee** [comment]
Orange alt confirmed

**Comment by Yenwodyah** [comment]
> Imagine a playerbase being this unrobust.

This doesn't have anything to do with war ops being hard. If anything, it's easier than stealth ops because it's harder for the ops to cheese. I just don't think that it's a good use of 50+ people's time to sit around for 20 minutes waiting for the ops to attack. It's like the end of every clock cult round all over again.

**Comment by necromanceranne** [comment]
What part of CQC, Emag and stimpack is hard. That's basically the hardest cheese there is.

**Comment by zxaber** [comment]
>I just don't think that it's a good use of 50+ people's time to sit around for 20 minutes waiting for the ops to attack. It's like the end of every clock cult round all over again.

Is it better for people to waste their time on the setup part of the round, only to be rolled by people armed to the teeth? I'd rather do setup for a fight that I know is coming in twenty minutes, rather than do setup for a round I think will last an hour and have that effort be wasted.

Stealth Ops is interesting content primarily for the Ops, security, and the captain. Everyone else might as well not exist.

**Comment by Spepbuthewranglesgithub** [comment]
can we just rework the entire gamemode. war ops is samey, blitz/stealth ops is the worst thing since sliced fusion, and trying to play the gamemode as its _theoretically_ intended leads to war ops but with less TC and no prep time for the crew.

**Comment by Yenwodyah** [comment]
> Is it better for people to waste their time on the setup part of the round, only to be rolled by people armed to the teeth? I'd rather do setup for a fight that I know is coming in twenty minutes, rather than do setup for a round I think will last an hour and have that effort be wasted.

So I think one of your concerns is well-founded - that stealth ops are pretty unbalanced and too powerful in their current state. My response there is the same thing that I said to wesoda - that stealth ops can easily be nerfed once people actually start playing them. Although I'm kind of feeling now that it might be better and more palatable to people to make some stealth ops nerfs and then pr them together with war ops removal. I might close this and try to do that after I hear what people say overnight. 
But in regards to your second concern, about having time wasted, I don't think that's any different from you getting attacked by a traitor 20 minutes into the round, or having your department blown up or something. Making an elaborate plan and executing it can be fun, but it's also fun to get thrown into a chaotic situation and have to make things up on the fly. We have extended and secret extended for the former, and nuke ops is for the latter. So, frankly, I think you should just accept that a lot of the time in this game you won't be able to execute perfectly what you're planning to do in the round.

**Comment by necromanceranne** [comment]
No amount of number tweaking is going to resolve the issue. It's that ops rely on alpha striking one guy and one guy only. To make that more difficult is to hobble the ops to the point of ineptitude.

At that point you might as well rework the mode from the ground up.

**Comment by Spepbuthewranglesgithub** [comment]
> No amount of number tweaking is going to resolve the issue. It's that ops rely on alpha striking one guy and one guy only. To make that more difficult is to hobble the ops to the point of ineptitude.
> 
> At that point you might as well rework the mode from the ground up.

exactly, traitors, cult, revs, wizards, and lings all get _most_ of the station involved, while nuke ops only need to interact with the disc holder, and can end the round a minute later.

**Comment by Yenwodyah** [comment]
> No amount of number tweaking is going to resolve the issue. It's that ops rely on alpha striking one guy and one guy only. To make that more difficult is to hobble the ops to the point of ineptitude.

Cult also depends on killing a single guy, the sacrifice. I don't think there's anything inherently unbalanceable about nukies.

**Comment by zxaber** [comment]
>But in regards to your second concern, about having time wasted, I don't think that's any different from you getting attacked by a traitor 20 minutes into the round, or having your department blown up or something. Making an elaborate plan and executing it can be fun, but it's also fun to get thrown into a chaotic situation and have to make things up on the fly. We have extended and secret extended for the former, and nuke ops is for the latter. So, frankly, I think you should just accept that a lot of the time in this game you won't be able to execute perfectly what you're planning to do in the round.

The issue isn't just about getting killed, it's that the round ends with nukies showing up, one way or another. Evading a traitor is nearly always possible if you pay attention and have a bit of luck, and even a work area being blasted to bits is just a setback if you're still alive and determined. But there's nothing in the game I can do to avoid the round-end screen when the last Op dies or the shuttle arrives. Once they get spotted, the round is over.

I'd much rather play the whole round knowing this.

**Comment by TheVekter** [comment]
This is going to be an unpopular change that gets merged anyway and the twenty or so people REEEEEing about it will shut up when they realize how much better things are without war ops.

**Comment by Spepbuthewranglesgithub** [comment]
> > No amount of number tweaking is going to resolve the issue. It's that ops rely on alpha striking one guy and one guy only. To make that more difficult is to hobble the ops to the point of ineptitude.
> 
> Cult also depends on killing a single guy, the sacrifice. I don't think there's anything inherently unbalanceable about nukies.

aaaaand that requires them to get atleast 9 people, that means setting up bases and capturing crewmembers that _can_ and most likely _will_ end up escaping, or calling you out first, or being seen, once someone does slip up the rest of the station is going to be on high alert meaning the chaplain is going to be raiding the armory and sec is going to be implanting/forcing holy water into everyone in a 5 mile radius, people will start to band together more, arm up, and start exploring maint, and eventually its going to go into all out war between the crew and the cult.

meanwhile by the time someone called out the ops the station is _probably_ already dead.

**Comment by Yenwodyah** [comment]
So your is basically with how the round always ends when the nukies die? How would you feel if the crew could choose whether to end the round or not after the nukies get killed? for example:
>centcomm announces "congrats crew, you killed the ops, we're going to send an expedited evac shuttle unless you decline"
>the crew has 2 minutes to decline/accept the shuttle at the comms console
>if they accept it or the time passes the round just ends immediately, they don't have to wait on the shuttle to get to centcomm
>otherwise they can continue the round and patch up the damage, maybe the round goes to mulligan

**Comment by Yenwodyah** [comment]
> meanwhile by the time someone called out the ops the station is _probably_ already dead.
Yeah, that's the kind of thing you can change with better balance - making it harder to beeline straight to the diskholder (e.g. removing emags from the op uplink), making it harder to be undetected (e.g. announcing the op shuttle like the pirate shuttle)

**Comment by zxaber** [comment]
Honestly, just a message about the Ops dying (like what happens with Revs during dynamic) would go a long way. It doesn't solve the issue with Ops not really engaging the entire crew (which War Ops fixes), but it does mean that the round can keep going if damage is minimal.

**Comment by necromanceranne** [comment]
> > No amount of number tweaking is going to resolve the issue. It's that ops rely on alpha striking one guy and one guy only. To make that more difficult is to hobble the ops to the point of ineptitude.
> 
> Cult also depends on killing a single guy, the sacrifice. I don't think there's anything inherently unbalanceable about nukies.

That's a false equivalency. There are several more steps involved in achieving a cult victory and they are not possible without the help of other members of the cult. A completely lone nuclear operative is all that is necessary to end the round.

Again, this isn't solved by just number tweaks. You need to repurpose what operatives have been sent to do, or you need to set up more steps for the nuclear operatives to achieve before the nuke can be activated and detonated. Hell, even taking out the portable nuke and replacing it with the onboard nuke makes it more akin to a defense point that needs to be held by the team and a congestion point for combat. Then make it more a countdown towards a point of no return AKA the nuke can't be defused anymore, and then give enough time for the nukies to evacuate themselves.

One gamer is going to have a far harder time detonating that nuke if everyone knows where it is and where they need to be to stop the nukies.

**Comment by Yenwodyah** [comment]
Well, there you go. That's a change you could make pretty easily to change the balance of stealth ops.

**Comment by XDTM** [comment]
Something like requiring multiple spread-out objectives (even better if they're different every time) would probably really help with these issues. Even better, the ops would then have the tactical choice to either split up to get everything they need more quickly (but if someone is taken down or discovered, they're in trouble) or move as a group for safety (but the crew can fortify the missing objectives).

**Comment by SkeletalElite** [comment]
All nuke ops needs is to have the state of things after the captain dies looked at. Like more of a chance for the crew to come back.

Like make it to where the nuke has a 2 or so minute "arming time" during which the disk has to stay in it and it can't be moved. When this "arming" initiates it sends an announcement with the nuke location.
Boom, blitz ops nerfed, war ops more fun since all the people who didn't get to do anything get another chance to fight in the last stand against the nukies.

**Comment by GuyonBroadway** [comment]
Wasn't this denied before?

In any case every argument made about warops can be made for stealthops only you don't get a warning klaxon and 20 mins to prepare. For every other player stealthops might as well be a min and a half long cutsceene before the station explodes, at least with warops you can try to do SOMETHING for 20 mins. If anything remove stealthops and you'd make them better lmao.


Heck Goon has the right idea. Plant the bomb disk or no disk, its location is announced and the crew can break it, applying disk halves its timer so getting the disk quicker is better. Add in a little window for ops to exfil post countdown and you got yerself a fun 'lil barrel of monkeys.

**Comment by ATH1909** [comment]
Making the ops need to use the nuke core instead of a portable nuke that they can hide in maint somewhere where no one will ever find it sounds like the best idea here. Currently, the round is pretty much over once the ops get the disk, as unless they're idiots and leave the disk in the nuke, the countdown timer's really just there for show/to give the ops time to evac.

**Comment by XDTM** [comment]
Maybe a middle way where they can call down a nuke via drop-pod into any major department, so the nuke defense doesn't get stale by being always in the same location.

**Comment by dee-dubya** [comment]
oh hey it's this PR again

can't wait to see people argue for a week or two, the PR get closed, and for things to continue as they should

**Comment by Tharcoonvagh** [comment]
*speedmerge pls*

**Comment by Michael-Ashfield** [comment]
I said this last time this PR came about,
There are tonnes to do in war OPS which normal players wouldn't get to do in their job, and the time pressure adds fun.

"Declaration of war":
- Virologist, oh crap! I better make a healing virus as fast as humanly possible and get it out to everyone, this is gonna be a challenge
- Engineer/atmos, I need to get as much power into the system AND build defences, maybe even a backup tcomms. I better make sure atmos is sabotage proof, we need as much pressure in distro as possible for when the pipes are blown, so there is still air to fill rooms.
- Botanist, Finally I can make murder plants, but I only have 20min, I better work hard.
- Doctor/paramedic, Better prep medbay, we are about to have tonnes of bodies flooding in, we need more surgery tables, more medicine, more spare body parts and more doctors!
- Chemist, should I make grenades or emergency meds, maybe a syringe gun...
- Scientist, we need max caps for points and to take out any OP warships around the station, hell we might be able to rush some good slimes to help us if we are fast and lucky, and what if I gave the crew healing nanites to revive them after death.
- Security, we don't have enough guns to go round for everyone, we. should have scouts looking out for the first sign of ops, and guards on the disk, release the prisoners and promise them a pardon if they kill a nukie.
- Assistants, I don't have much but I can make spears, use welder bombs and scavenge maint for useful weapons and tools, perhaps I can form a group to rush any nukies as soon as they land.

Basically, if you're standing around twiddling your thumbs as crew, you're playing nuke OPs wrong or just not doing enough for the station. I do agree that OPs can be bored for a little of the 20min, but then they do immediately get the most fun of all players that round straight after the tiny wait.

**Comment by trollbreeder** [comment]
deja vu
might as well delete all gamemodes except traitor, ling, traitorling and extended because muh autism projects
that includes deleting revs which arguably is even more of a TDM except one team can outrecruit the other in a matter of minutes and the other team has 3 more guys they need to protect
#45307

**Comment by HomocidlePancakes** [comment]
What a fucking pathetic excuse of a human being. Not every round should play out how you want it to be. Just wait until the next round instead of throwing tantrums and making an absolute clown out of yourself.

**Comment by XDTM** [comment]
The game shouldn't guarantee your success in everything you attempt, but it's understandable that someone would get frustrated if they have limited time to play and the rng says that they get to play team deathmatch instead of rp spacemen.

That said, regular nuke ops is even worse, since it cuts the round short _after_ you put in the effort to start whatever you were planning.

**Comment by boy2mantwicethefam** [comment]
Stop playing on bad servers

**Comment by actioninja** [comment]
I am working a rework that would make the concept of war ops obsolete and remove it by extension.

**Comment by MrDoomBringer** [comment]
>I said this last time this PR came about,
>There are tonnes to do in war OPS which normal players wouldn't get to do in their job, and the time pressure adds fun.

>"Declaration of war":

>Virologist, oh crap! I better make a healing virus as fast as humanly possible and get it out to everyone, this is gonna be a challenge
Engineer/atmos, I need to get as much power into the system AND build defences, maybe even a backup tcomms. I better make sure atmos is sabotage proof, we need as much pressure in distro as possible for when the pipes are blown, so there is still air to fill rooms.
Botanist, Finally I can make murder plants, but I only have 20min, I better work hard.
Doctor/paramedic, Better prep medbay, we are about to have tonnes of bodies flooding in, we need more surgery tables, more medicine, more spare body parts and more doctors!
Chemist, should I make grenades or emergency meds, maybe a syringe gun...
Scientist, we need max caps for points and to take out any OP warships around the station, hell we might be able to rush some good slimes to help us if we are fast and lucky, and what if I gave the crew healing nanites to revive them after death.
Security, we don't have enough guns to go round for everyone, we. should have scouts looking out for the first sign of ops, and guards on the disk, release the prisoners and promise them a pardon if they kill a nukie.
Prisoner, now is my chance to escape, I'm gonna make sure these security goons never see me again, hell they even handed me a spear!
Assistants, I don't have much but I can make spears, use welder bombs and scavenge maint for useful weapons and tools, perhaps I can form a group to rush any nukies as soon as they land.
The lack of paranoia is a fun mechanic sometimes as you can truly see how much the station can achieve when working together with a tight time limit, it leads to interesting and unique interactions and gameplay.

>Basically, if you're standing around twiddling your thumbs as crew, you're choosing to miss out on high-pressure defence building and unique interactions with crew who are also scrambling to protect the station. I do agree that OPs can be bored for a little of the 20min, but then they do immediately get the most fun of all players that round straight after the tiny wait.

based comment
also
\>OP complaining about every war ops round being too similar to one another
\>OP then says that war op rounds should be more similar to every other kind of round we have
\>OP lobbies for more roleplaying/paranoia then says he would prefer a ton of incredibly short wizards rounds??
\>OP doesnt understand the concept of game modes giving players a chance to focus on different aspects of the game

yup this ones going in my cringe comp

**Comment by Iamgoofball** [comment]
why play war ops when you can play Families?

**Comment by Yenwodyah** [comment]
@Michael-Ashfield I don't think anyone's saying that there's _nothing to do_ on war ops, it's just that you're limited to doing the _exact same thing_ every round. It's an incredibly repetitive gamemode.

First - most of the things you listed are things people can do on normal rounds too. 
 - Virologists can always make healing viruses. 
 - Chemists can always make grenades and healing mixes.
 - Scientists always rush maxcaps. 
 - I don't think I've ever seen an engie build a backup tcomms, even on nukies, but that's something that they could do on any round, and would be useful on more than just war ops too (whenever the malf AI decides to turn off comms, for instance...)

If you're seeing these behaviors only on war ops, I think that's less because war ops allows for more types of gameplay and more because _there's nothing else to do_ on war ops. 
 - Is the chemist going to  bother making an industrial-scale colored lube factory? No, what's the point? The round will end and everyone else is focused on killing the ops anyways. 
 - Is the engineer going to set up the singulo or the TEG? Round will probably be over by the time we have the parts, so it's pointless. 

War ops doesn't _allow_ players to experience this gameplay, it _forces_ them to. It might be fun the first few times you go through it, but after you've played the game for a while, it gets reeeeally stale.

(Stealth ops has this problem too, but like I said in previous comments, I think that's something we can change by nerfing and tweaking the gamemode. And, fuck, when stealth ops get called out and you realize the round's going to end, at least you only have to wait like 10 minutes for the next round instead of 30.)

**Comment by ATH1909** [comment]
Why not, just, like, disable (war) ops on Manuel, but keep it on the LRP servers?

**Comment by BadSS13Player** [comment]
> Why not, just, like, disable (war) ops on Manuel, but keep it on the LRP servers?

The PR that this one is trying to close makes warops a config.

**Comment by ExcessiveUseOfCobblestone** [comment]
Gonna DNM this until the config PR is done.

This, like the other gamemodes, needs to be disabled before entertaining it from removing the code imo unless there's an issue that affects the code. This is coming from someone who s* everytime the announcement is made.

This means either the other PR getting merged then warops getting disabled OR ops in general being disabled now.

Would also accept a rework of ops that phases out the need for a gamemode to have a button that 180's the play style of said mode.

**Comment by GuyonBroadway** [comment]
Warops isn't a 180 on the nukie playstyle. Flucops are tactically felxible and allow for multiple playstyles with each operative getting to choose thier style. warops is simply doubling down on a specific one and having the whole team follow it. If someone could figure out how to code it, you'd bet there would be a "declare stealth" button.

Only change I would make to warops is A, allow it to be done regardless of pop for a scaling amount of tc and/or B make it so the team needs to vote for warops and not just at the discretion of the leaderm

**Comment by Timonkeyn** [comment]
imagine you getting good discounts for a nice traitor gimmick, taking your time to strike only for a delta alert to pop up 15 minutes in.

**Comment by ExcessiveUseOfCobblestone** [comment]
evil oranges

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> Warops isn't a 180 on the nukie playstyle. Flucops are tactically felxible and allow for multiple playstyles with each operative getting to choose thier style. warops is simply doubling down on a specific one and having the whole team follow it. If someone could figure out how to code it, you'd bet there would be a "declare stealth" button.
> 
> Only change I would make to warops is A, allow it to be done regardless of pop for a scaling amount of tc and/or B make it so the team needs to vote for warops and not just at the discretion of the leaderm

a wide variety of tactics and such being turned into beeline murderbone TDM seems like a 180 to me but I get what you're saying and agree.

**Comment by AdamElTablawy** [comment]
frankly if you hate the basis of war ops you'll quickly find regular ops just as unfun because stealth has no counter and blitz is war ops but everyone dies because they don't have guns

just go with goon ops at this point

**Comment by zxaber** [comment]
Yenwodyah's post sorta nails it on the head for me. Deathmatch or not, Warops pulls the entire crew into the event. Every job that can be weaponized shifts gears to prep the fight, and those that can't either arm up and bunker down or do what they can to prepare for cleanup/repair duty.

 And that's a hell of a lot more interesting than "another day at the office but then it explodes because a dude you've never met lost to a dude in a red suit".

**Comment by dootdoom** [comment]
clown ops should still be able to declare war because they're clowns and it's funny haha r-right?

**Comment by Yenwodyah** [comment]
> And that's a hell of a lot more interesting than "another day at the office but then it explodes because a dude you've never met lost to a dude in a red suit".

Here's the thing: if you've played it more than, like, ten times, war ops _is_ another day at the office. Every war ops round is functionally identical to every other one. I would much rather have 20 minutes with the _possibility_ of something interesting happening, then the station explodes, than a whole round in which I _know_ nothing new or exciting or innovative is going to happen.

On stealth ops, I at least feel surprise and panic when the delta alert starts. I think "oh shit, I gotta get out of here NOW". On war ops, I just think "oh god, not this again".

**Comment by wesoda25** [comment]
love war ops bc i can always look forward to the same carbon copy of a round that ive sat through the past 3 years

**Comment by SkeletalElite** [comment]
> On stealth ops, I at least feel surprise and panic when the delta alert starts. I think "oh shit, I gotta get out of here NOW". On war ops, I just think "oh god, not this again".

On stealth ops I start doing whatever it I try to do and sigh in disappointment when I see the delta alert because it literally does not matter what you do the round is going to be over in a minute.
On war ops I think, cool I get to weaponize my job and use it to try and robust some nukies.

**Comment by Michael-Ashfield** [comment]
> I would much rather have 20 minutes with the possibility of something interesting happening, then the station explodes

So you want to replace 20min of unique and interesting defence building/planning where crew can do stuff they never usually can (even as antag) as a team, followed by some of the most intense and engaging fights SS13 has to offer where the crew feel involved in the outcome of the round.
with...
20min of the same old round where there is a chance that the clowns' gimmick will be interesting, followed by a random delta alert and collective sigh?

I mean, you are even taking away fun for the OPs themselves.

**Comment by XDTM** [comment]
> So you want to replace 20min of unique and interesting defence building/planning 

Unique compared to other modes, but every war ops round is very similar to other war ops rounds, since the best tactics eventually become commonplace. Give all access to everyone, open the armory, give the disk to the captain or the clown and fortify a location, and so on.

> where the crew feel involved in the outcome of the round.

Unless your job happens to have no combat utility or takes more than 20 minutes to reach it.  Then you might as well suicide and spectate to maybe at least get some enjoyment out of the round.

Yes, normal ops sucks as well for the crew, but war ops isn't much better.

**Comment by BadSS13Player** [comment]
> Yes, normal ops sucks as well for the crew, but war ops isn't much better.

They are much better. If you hate warops so much, at least you can suicide, otherwise you just have a round that ended too soon and nothing you could have done about it.

**Comment by Michael-Ashfield** [comment]
>Unless your job happens to have no combat utility or takes more than 20 minutes to reach it. Then you might as well suicide and spectate to maybe at least get some enjoyment out of the round.

If you're not helping the station or getting involved in some capacity, that's on you as a player, not the game mode. Every role is capable of helping the station insome way, including assistants. 
The presence of nukies simply gives you more freedom on how you go about doing your role to defend the station.

**Comment by XDTM** [comment]
> 
> 
> > Unless your job happens to have no combat utility or takes more than 20 minutes to reach it. Then you might as well suicide and spectate to maybe at least get some enjoyment out of the round.
> 
> If you're not helping the station or getting involved in some capacity, that's on you as a player, not the game mode. Every role is capable of helping the station in some way, including assistants.
> The presence of war nukies simply gives you more freedom on how you go about doing your role to defend the station.

True, but proper gamemodes allow each job to serve their function during the round in one way or another, instead of turning some into fancy-dressed assistants.

**Comment by Michael-Ashfield** [comment]
>True, but proper gamemodes allow each job to serve their function during the round in one way or another, instead of turning some into fancy-dressed assistants.

"You are the curator"
;GUYS ITS WIZARD!!
*become battle assistant*

"Your are the chaplain"
;GUYS ITS REVS!!
*become battle assistant + armour*

"You are the bartender"
;GUYS ITS CULT!!
*Become battle assistant + shotgun*

"You are the mime*
;The captain is being beaten to death, I saw red  hardsuits, I think it is nukies
*Become silent battle assistant*
"Delta alert"


I mean like, other rounds offer less compared to war nukies as it allows you to do your job and prep, 20min is long enough for every non-RP job to do something productive, and plenty for RP jobs to get a gimmick going.

**Comment by GuyonBroadway** [comment]
Name a job that CAN'T help in warops and I'll tell you why you are wrong.

**Comment by XDTM** [comment]
I don't mean it can't ever be fun, nor that all other gamemodes are perfect (i personally have a gripe against the mentioned gamemodes in particular). 

Really, what bothers me about war ops could probably be solved if it simply happened later in the round (probably as a result of dynamic?). That way, jobs would have settled in for, say, 30 minutes; at that point the station would be in a state that could vary wildly from round to round. 
Once the war announcement comes, then:
- The jobs that were engaging against roundstart war ops still are; assistants, sec and captain and anyone who joins in can fight.
- Anything that takes 30-50 minutes is now on the table, bringing along more unusual weapons and tactics to increase diversity between rounds. Mining loot, slime stuff, advanced surgery enhancements and nanites, and so on.
- The station could be in any state, either running like clockwork and quickly preparing their defenses, or an apocalyptic hellhole with several singularities running amok. It might not result in perfect balance, but it will make war ops rounds feel different from each other
- RP jobs are still kinda shafted, but they get 30 minutes of normal round beforehand at least

Basically, increasing variance between rounds (a common complaint about the gamemode) by letting the station evolve a bit before pulling the trigger.

**Comment by GuyonBroadway** [comment]
To be fair, a lot of gamemodes could be improved by letting the station play out a while before dropping antags.

**Comment by Geikkamir** [comment]
Honestly Nuke Ops in general would be improved massively if there were just more objectives that getting the disk. If nukes got 3 or 4 other steal/sabotage objectives it would go a long way towards making the strategy more interesting on both sides than "throw literally everything at the captain."

**Comment by Shadowflame909** [comment]
> Name a job that CAN'T help in warops and I'll tell you why you are wrong.

Clown and Lawyer

**Comment by trollbreeder** [comment]
> Nukies get boslips

Newfriend, have you ever given space lube to a clown before?

**Comment by HomocidlePancakes** [comment]
Why is this still open?

**Comment by GuyonBroadway** [comment]
>Clown and Lawyer

That pie cannon is a mighty robust device in the right hands and the lawyer has implants and brig access putting you on the top of the list for the "free guns militia". Plus you get a laser pointer if I am not mistaken which can fuck over syndieborgs.

**Comment by Fikou** [comment]
> Why is this still open?

because my friend orange really doesnt like war ops

**Comment by MrDoomBringer** [comment]
@Yenwodyah How would you feel about letting nukies choose between stealing the nuke disk and, say, a sliver of the SM crystal when they declare war? With the war declaration message alerting the crew what they need to protect.

**Comment by Yenwodyah** [comment]
@MrDoomBringer So basically a stationary objective instead of a mobile one? I think that would shake things up and make war ops a little more interesting. If you PR it I'll give you an upthumb.

**Comment by necromanceranne** [comment]
@MrDoomBringer something like the SM sliver just means the crew blows the SM in order to prevent the nukies winning at all. Barring that, an accidental delamination is still going to be difficult to ever prove as an act of metagaming, and still denies the nukies victory.

Nuke disk is station loving and indestructible for a reason.

**Comment by YakumoChen** [comment]
imagine if the ops had to go deep cover to do their objective instead of just running in with a SAW and mowing down half the station?

**Comment by GuyonBroadway** [comment]
>imagine if the ops had to go deep cover to do their objective instead of just running in with a SAW and mowing down half the station?


It exists, it's called syndicate infiltrators

**Comment by MrDoomBringer** [comment]
@necromanceranne If we added the SM crystal as an objective, I think it'd be natural that we'd give the crew the goal of securing it as well. Blowing up the SM would fail the round for both teams. Perhaps the emergency shuttle would come with a SM-crystal extraction device the crew could use.

**Comment by Spepbuthewranglesgithub** [comment]
wait, this still hasn't been merged yet? i thought this would have been over with already.

_can someone just merge this already so ops can finish devolving into 15 minute round: the gamemode_

**Comment by dee-dubya** [comment]
> 
> 
> > Clown and Lawyer
> 
> That pie cannon is a mighty robust device in the right hands and the lawyer has implants and brig access putting you on the top of the list for the "free guns militia". Plus you get a laser pointer if I am not mistaken which can fuck over syndieborgs.

lawyer does not start with an implant, they get brig access and security comms

you're right that brig access naturally gives them a head start on the gun rush though, assuming armory gets bolted open rather than people swarming it with their AA

**Comment by ShizCalev** [comment]
image

**Comment by MrDoomBringer** [comment]
the savior /tg/ needed

**Comment by GuyonBroadway** [comment]
I think this is the first time I've seen a PR closed due to community feedback and downstoats

**Comment by Fikou** [comment]
> I think this is the first time I've seen a PR closed due to community feedback and downstoats

the pr that removed the stun bolt from charliestation energy gun was closed from downdoots too

**Comment by ExcessiveUseOfCobblestone** [comment]
next month we'll get 'em...

**Comment by GuyonBroadway** [comment]
How to merge an unpopular and unwanted PR.

1: make initial pr.
2: let it get closed cus it's a dumb idea
3: wait 3 months
4: open the exact same pr again
5: when people bring up the reasons it was closed last time disregard them as "the same tired arguments despite not assessing them in the first place.

**Comment by Shadowflame909** [comment]
A 3 strike PR rule would be good. If someone keeps trying to remove a subject over and over and over, and it fails 3 times. Then they shouldn't be allowed to work on anything pertaining to it anymore.
